app/views/general_mailer/monthly_report_message.html.haml

```diff
@@ -0,0 +1,9 @@
+%html
+  %head
+  %body
+    %h3
+      NITRO Competitions Report for the previous month.
+      = Date.today
+    %p
+      Please find attached a csv file with information for competitions that were created or updated during the previous month. 
+    
```

Comment:
Just a formatting nitpicking - I'd keep newlines at the end of the file, it improves readability